.github/CONTRIBUTING.md

```diff
@@ -154,15 +154,158 @@ the repo such that they can get coding -->
+They also provide helpful checklists that helps you follow [the process (INSERT PROPER LINK HERE)](#the-process) and lets others know how far you are. 
+
+If you experience that no template lives up to your specific need; well there is a good opportunity to contribute. Create an issue for a new template to be created! 
+
+#### The good issue has a good title
+<!-- - Spend some time on your titles 
+  - Make them alluring. 
+  - Short without too much detail 
+  - The issues are viewed in some sort of list view 80% of the time. -->
+As issues are viewed in some sort of list view 80% of the time (or so) the title is the part being read most often. 
+It can therefore be a good idea to spend some time formulating one. 
+
+A good title is short without too much detail. Details should be presented in the actual description. 
+
+If possible try to make the title compelling. While an open-source repository is not a newspaper where readers must be enganged at any cost, you still want to catch the interest of contributors. 
+
+The following title: 
+> Input causes error when you type over 50 characters and press enter 
+
+could be improved by writing: 
+> Input error on 50-character submit
+
+#### The good issue is easily readable
+<!-- - Readability 
+- Easily understandable 
+  - No internal lingo 
+- Proof-read --> 
+
+There are MANY ways to improve the readability of a text. The following is just some ways to do it, that we think can add value when writing an issue:
+- Using proper punctuation (to the best of your abilities at least. We are not experts at ,it either)
+- Spending some time proof-reading the issue before it is submitted. 
+- Using available formatting such as lists and code blocks to aid easy skimming. 
+- Explaining the abbrevations being used 
+
+<!-- Perhaps come up with a more code related example - i just couldn't at the time of writing -->
+For example the text: 
+> I went to the RDL the other day, i think that the RDL is very nice! They have a huge selectin of books such as: the old man and the sea, 1984, Brave new world and the count of monte cristo 
+
+Is much harder to read than: 
+> I went to the royal danish library (RDL) the other day. The RDL is very nice. They have a huge selection of books, such as: 
+> - the old man and the sea 
+> - 1984 
+> - brave new world 
+> - the count of monte cristo
+
+#### The good issue contains screenshots and gifs if possible 
+<!-- - A picture says more than 1000 words... (Imagine how much a gif is saying) --> 
+Kirby is a design system, hence it is a very visual project! So do not be shy of using images and gifs to explain yourself. 
+
+If you are requesting a new component, perhaps add a sketch showing how you are imagining it. When reporting a bug, record a gif to show exactly what you are experiencing. 
+Suggesting an enhancement? Add before and after pictures portraying what you would like. 
```

Comment:
Maybe "screen recording" or "screen capture" instead of "gifs" - personally I use video way more often than gifs (never use them 😊).

Anyway,  my point is that we shouldn't ask contributors to provide gifs if we prefer video or screenshots. So to me "screen recording" or "screen capture" are better words.

Reply:
I've changed it to "screen recordings". 

I've seen "screen capture" used as a synonym for "screenshot" before, so I will abstain from using that. 

![Cool Gif](https://media2.giphy.com/media/l0Exx2lopTC4mJaP6/giphy.gif?cid=ecf05e47xrbxck5tvc9jjkpgnsptun0o874gk2dab3tdt6m3&rid=giphy.gif&ct=g)

Reply:
https://user-images.githubusercontent.com/150451/115716187-36c23680-a379-11eb-9213-c164925eb1cb.mov